### DISCUSSION TITLE
streaming mod

### DISCUSSION THREAD

**Discussion by wollframium** [JustArchiNET/ASF-ui#1696] [Ideas]
I don't know if there are any streamers among Archie users (I'm not a streamer myself), but I think it would be useful to make a toggle switch (without rebooting ASF) in which the bot nicknames will either be hidden or changed to bot1 bot2 and so on, or blurred, so as not to show the bot nicknames on the stream and viewers do not file a complaint.
do this on these pages
http://127.0.0.1:1242/bots
http://127.0.0.1:1242/commands
and also burn the nicknames can be compromised if you click on a specific bot and the address will contain
http://127.0.0.1:1242/bot/botnickname
here you need to assign a number to the bot in case of streaming

**Comment by wollframium** [comment]
How do I see it
http://127.0.0.1:1242/bots
![scren](https://github.com/user-attachments/assets/ffee48e1-a681-44ba-bb2b-e742c0304a18)
http://127.0.0.1:1242/bot/botnick
![Снимок2](https://github.com/user-attachments/assets/fecfcfe1-dfd5-4348-88b8-a9c6a6fef628)
http://127.0.0.1:1242/commands
![Снимок](https://github.com/user-attachments/assets/4bb2ebd9-cac9-4176-9a35-74459388c29e)

**Comment by JustArchi** [comment]
Bots are identified by names, not nicknames, and you have full control over how you want to name your bots, since they're based on file names. Bot name by definition is not a sensitive detail since it has no correlation with any login credential, password or anything else.

If you decided yourself to name bots "inappropriately", then that's on you, it's not like you can't change it. I don't see any value of this suggestion to be honest, but I'm not ASF-ui dev so I'm not making any call here.

**Reply by Aareksio** [reply]
I agree this is out of scope, but maybe @MrBurrBurr thinks otherwise
